### PR TITLE
updated robertson docs

### PR DESCRIPTION
**Pull request by elswit**

### CODE REVIEW

File: toolbox/+otp/+robertson/+presets/Canonical.m [hunk 1]
```diff
@@ -1,5 +1,5 @@
 classdef Canonical < otp.robertson.RobertsonProblem
-    % The classic problem with the classic coeeficients
+    % The classic problem with the classic coeeficients $K_1 = 4e-2, K2 = 3e7, K3 = 1e4$.
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/+presets/Canonical.m:2]: Mention this preset is referred to as ROBER in Solving ODEs II pg 144.

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 2]
```diff
@@ -1,35 +1,77 @@
 classdef RobertsonProblem < otp.Problem
-    %ROBERTSONPROBLEM A simple, stiff chemical reaction
-    %   This problem models the concentration of chemical species A, B, and C 
-    %   with the reactions
+    % The Robertson Problem is  a simple, stiff chemical reaction that
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:2]: Need a one line summary before the description

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 3]
```diff
@@ -1,35 +1,77 @@
...
+    % models the concentrations of chemical species A, B, and C 
+    % involved in the reactions
+    % $$
+    % \begin{align*}
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:6]: I don't think align is needed

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 4]
```diff
@@ -1,35 +1,77 @@
...
-    %   These correspond to the ODEs
+    % 
+    % Notes
+    % -----
+    % +---------------------+-------------------------+
+    % | Type                | ODE                     |
+    % +---------------------+-------------------------+
+    % | Number of Variables | 3                       |
+    % +---------------------+-------------------------+
+    % | Stiff               | yes                     |
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:34]: yes -> typically, depending on $K_1$, $K_2$, and $K_3$.

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 5]
```diff
@@ -1,35 +1,77 @@
...
-    %   which makes the problem challenging. This has made it a popular test for
-    %   implicit integrators.
-    %
-    %   Sources
-    %
-    %   
-    %
-    %   See also otp.robertson.RobertsonParameters
-    
+    %
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:41]: Remove empty line

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 6]
```diff
@@ -1,35 +1,77 @@
...
-    %   See also otp.robertson.RobertsonParameters
-    
+    % 
+    % >>> p = otp.robertson.presets.Canonical;
+    % >>> problem.Parameters.K1 = 100;
+    % >>> problem.Parameters.K2 = 1000;
+    % >>> problem.Parameters.K3 = 1000;
+    % >>> sol = p.solve();
+    % >>> p.plot(sol)
+    %
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:48]: remove empty line

File: toolbox/+otp/+robertson/RobertsonParameters.m [hunk 7]
```diff
@@ -1,11 +1,11 @@
 classdef RobertsonParameters
-    %ROBERTSONPARAMETERS User-configurable parameters for the Robertson problem
-    %
-    %   See also otp.robertson.RobertsonProblem
-    
+    % Parameters for the Robertson problem.
     properties
+        % Reaction rate $K_1$
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonParameters.m:4]:
```suggestion
        % The reaction rate $K_1$
```

File: toolbox/+otp/+robertson/RobertsonParameters.m [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
         K1 %MATLAB ONLY: (1, 1) {mustBeReal, mustBeNonnegative}
+        % Reaction rate $K_2$
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonParameters.m:6]:
```suggestion
        % The reaction rate $K_2$
```

File: toolbox/+otp/+robertson/RobertsonParameters.m [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
         K2 %MATLAB ONLY: (1, 1) {mustBeReal, mustBeNonnegative}
+        % Reaction rate $K_3$
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonParameters.m:8]:
```suggestion
        % The reaction rate $K_3$
```

File: toolbox/+otp/+robertson/RobertsonProblem.m [hunk 10]
```diff
@@ -51,4 +93,4 @@ function onSettingsChanged(obj)
                 varargin{:});
         end
     end
-end
+    end
```
Comment (Steven-Roberts) [inline toolbox/+otp/+robertson/RobertsonProblem.m:96]:
```suggestion
end
```

**Review by Steven-Roberts — CHANGES_REQUESTED** [review]

File: toolbox/+otp/+robertson/RobertsonProblem.m (hunk 3 above)
Comment (elswit) [reply toolbox/+otp/+robertson/RobertsonProblem.m:6]: It's tider when $K$s are aligned

**Review by elswit — COMMENTED** [review]
fixed md table miss align

**Pull request merged**